devlib/native/win_native.cpp

```diff
@@ -304,18 +306,17 @@ namespace winutil {
     }
 
 
-    static auto getDevInfo(const QString containerId) -> DevInfo {
-        DevInfo devInfo;
+    static auto getUsbDevicesByContainerIdsMap(void) -> QMap<QString, DeviceProperties> {
```

Comment:
I suggest naming this method as `getMapOfUsbDevicesByContainerIds` and the variable could be left unchanged

Reply:
Changed